scripts/ctool

```diff
@@ -562,6 +563,7 @@ call_in_env() {
             -u|--user) user="$next"; shift;;
             --user=*) user="${cur#*=}";;
             -d|--dir) dir="$next"; shift;;
+            --def-cmd=*) defcmd="${cur#*=}";;
```

Comment:
is -def-cmd surfaced to ctool ?

Reply:
no. It probably would get handled. but I woudnt bother.  It really was just required to get 'call_in_env' to use that as the default command .  the caller (execute) didnt have an easy way to see if there was or was not a command provided without more involved parsing.